website/docs/index.html.markdown

```diff
@@ -251,6 +251,8 @@ for more information about connecting to alternate AWS endpoints or AWS compatib
   - [`aws_instance` data source](/docs/providers/aws/d/instance.html)
   - [`aws_instance` resource](/docs/providers/aws/r/instance.html)
   - [`aws_launch_template` resource](/docs/providers/aws/r/launch_template.html)
+  - [`aws_qldb_ledger` data source](/docs/providers/aws/d/qldb_ledger.html)
```

Comment:
This data source and resource do not need to be added to the `skip_requesting_account_id` documentation because they are not manually creating their own ARNs from the provider account ID.